contents/blog/vc-or-bootstrap.md

```diff
@@ -0,0 +1,101 @@
+
+There are lots of myths around what it's like run a business with VC funding - today, I'm here to dispel them.
+
+## Do board meetings with VC companies involve them pressuring expansion at all costs?
+
+Not for us.
+
+This depends entirely on who joins your board, and how you've pitched to them.
+
+Our experience has been an intense focus on finding the Ideal Customer Profile, and trying to focus as much as possible on the right users. Yes, we track revenue, signups and so on, but our board sees those as the output of building a great product for a specific set of end users. 
+
+We have to prepare for the board meetings - now that we've got the process down, these usually take a few hours to do. We keep the majority of the meeting focused on one or two discussion topics rather than presenting an update. The guidance our board have given us has outweighed the cost of setting up and running these sessions every 3 months. And, frankly, if I know someone is marking my homework, it encourages me to do a better job. Sidenote: is this the real reason we're open source?!
+
+## Control
+
+Our board could pressure us to take certain decisions, but Tim (cofounder) and I are ultimately in control - after our series B, we've got control of three out of five seats. Control is something we can negotiate each time we fundraise.
+
+The fact we have plenty of runway, can pay everyone well, and have great people that ship stuff, means we have _more_ control, than a bootstrapped startup that _must_ generate revenue or go out of business.
+
+Of course, this can easily result in a lack of discipline. However, it has helped us be able to get an open source project off the ground first, before focussing on revenue last summer.
+
+## Is fundraising a drag?
+
+There is a huge amount of variation. It all comes down to roughly these requirements:
+
+* clear explanation of what you're working on
+* fast progress being made
+* an idea that could be very big
+* proven team
+* good vc market conditions
+
+I've intentionally not listed "a good network" as, whilst easier for some than others, I believe you can get this done as a byproduct of the above.
+
+None of these are necessary, but the better you do against each of them, the less time you'll have to spend fundraising. Ironically, this means you'll have more time to focus on product, which means - the easier time you'll have fundraising.
+
+## More bosses
+
+Although you may retain control, if you fundraise, you've now got investors that you feel you _should_ report to.
+
+However, if you're building a bootstrapped company, you _still_ have to report to someone - your own team, if applicable, and your customers.
+
+One more party has made a positive difference to us, but for others it can be bad - if your board asks for X, and users want Y, then you may have problems. This depends on the kind of people you raise from and your relationship with them.
```

Comment:
```suggestion
One more party has made a positive difference to us, but for others it can be bad – if your board asks for X, and users want Y, then you may have problems. This depends on the kind of people you raise from and your relationship with them.
```